### ISSUE TITLE
compare hyper-parameter settings between domainbed benchmark and domainlab benchmark in examples yaml file

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#210]
the domainbed has a benchmark here: https://github.com/facebookresearch/DomainBed, while our current hyper-parameter settings for algorithms are https://github.com/marrlab/DomainLab/blob/master/examples/yaml/benchmark_pacs.yaml
to make a fair comparison, it is important to compare as many shared hyper-parameters (if possible).  

deliverable: 
1. search the paper of domainbed and find out where the hyper-parameters and experimental settings were specified
2. analyze some key difference: e.g. optimizer (which optimizer were used, with what learning rate, are there learning rate schedule?
3. it is definitely not possible to make a 100% comparison as we have many implementation details that differ from domainbed, we just need to get a partial picture of the difference and try to align as much as possible.
